erpnext/selling/doctype/sales_order/sales_order_list.js

```diff
@@ -351,8 +351,56 @@ frappe.listview_settings['Sales Order'] = {
 			}
 		};
 
+		const create_one_production_plan_action = () => {
+			const selected_docs = listview.get_checked_items();
+			const docnames = listview.get_checked_items(true);
+
+			if (selected_docs.length > 0) {
```

Comment:
```suggestion
			if (selected_docs.length === 0) {
```
- Reduce the indentation

Reply:
> * Reduce the indentation

Whole execution is performed inside this block and the error thrown after this is for  for **On Hold** and **Closed** Orders.

Reply:
If the length is equal to zero, do not perform it, just wanted you to reduce the indentation
```suggestion
			if (selected_docs.length === 0) { return; }
```

Reply:
@hrwX I think the current indentation seems more intuitive.